src/main/java/sk/uniba/fmfi/controller/BodyValuesCalculator.java

```diff
@@ -1,22 +1,65 @@
 package sk.uniba.fmfi.controller;
 
 import lombok.Data;
+import lombok.NonNull;
+import lombok.SneakyThrows;
 import sk.uniba.fmfi.model.BodyRecord;
+import sk.uniba.fmfi.model.UserInfo;
+
+import java.util.InputMismatchException;
```

Comment:
Tento import sa nepoužíva. Skontroluj aj ostatné triedy a tiež odstráň nepoužité importy.

Reply:
fixed